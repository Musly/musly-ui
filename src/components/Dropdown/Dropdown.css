.dropdown {
  position: relative;
  cursor: pointer;
  user-select: none;
}
