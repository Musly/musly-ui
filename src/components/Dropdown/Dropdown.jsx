import React from 'react';
import PropTypes from 'prop-types';
import './Dropdown.css';

export function Dropdown ({ children }) {
  return (
    <div className="dropdown" data-test-id="dropdown">
      {children}
    </div>
  );
}

Dropdown.propTypes = {
  children: PropTypes.node.isRequired,
};
