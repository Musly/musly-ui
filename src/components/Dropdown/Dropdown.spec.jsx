import React from 'react';
import { shallow } from 'enzyme';
import { Dropdown } from './Dropdown';

describe('<Dropdown />', () => {
  it('should render as expected', () => {
    const result = shallow(<Dropdown><div>Test</div></Dropdown>);
    expect(result.find('[data-test-id="dropdown"]')).toExist();
    expect(result).toMatchSnapshot();
  });
});
