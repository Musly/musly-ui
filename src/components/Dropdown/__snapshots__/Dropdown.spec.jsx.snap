// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dropdown /> should render as expected 1`] = `
<div
  className="dropdown"
  data-test-id="dropdown"
>
  <div>
    Test
  </div>
</div>
`;
